exercises/practice/affine-cipher/.meta/config.json

```diff
@@ -18,10 +18,8 @@
     "test": [
       "affine_cipher_test.rb"
     ],
-    "example": [
-      ".meta/solutions/affine_cipher.rb"
-    ]
+    "example": ".meta/example.rb"
   },
   "source": "Wikipedia",
   "source_url": "http://en.wikipedia.org/wiki/Affine_cipher"
-}
+}
```

Comment:
A `JSON` file should end with an EOL as it is a text file.

Reply:
Aha, I did not know that.  Fixed.
`configlet lint` now passes locally (but with warnings I think are outside the scope of this fix ).